src/xcode/ENA/ENA/Source/Scenes/ErrorReporting/MainErrorReportViewControllers/TopErrorReportViewModel.swift

```diff
@@ -19,8 +23,15 @@ class TopErrorReportViewModel {
 		var dynamic = DynamicTableViewModel([])
 		dynamic.add(
 			.section(cells: [
-				.body(text: AppStrings.ErrorReport.description1),
-				.link(text: AppStrings.ErrorReport.faq, url: URL(string: "https://example.com"), accessibilityIdentifier: AccessibilityIdentifiers.ErrorReport.faq) // TO DO: get correct link!
+				.body(
+					text: AppStrings.ErrorReport.description1,
+					accessibilityIdentifier: AccessibilityIdentifiers.ErrorReport.topBody
+				),
+				.link(
+					text: AppStrings.ErrorReport.faq,
+					url: URL(string: "https://example.com"), // TO DO Get correct Link
```

Comment:
There is a link available: https://www.figma.com/file/jYFqboxVmVDOM8S7sFggw2?node-id=4752:25485#64933782